pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -164,6 +164,15 @@ sys.exit('no disk found with OSD ID $OSD_ID')
 	fi
 	[[ -z "$DEVICE" ]] && { echo "no device" ; exit 1 ; }
 
+	# If a kernel device name change happens and a block device file
+	# in the OSD directory becomes missing, this OSD fails to start
+	# continuously. This problem can be resolved by confirming
+	# the validity of the device file and recreating it if necessary.
```

Comment:
The re-creating of the path is done by ceph-volume, correct?

Reply:
It's correct.